Frontend/ui-customer/src/Components/SearchLocationModel/View.purs

```diff
@@ -485,6 +486,47 @@ recenterButtonView push state =
         ]
   ]
 
+findPlacesIllustration :: forall w. (Action -> Effect Unit) -> SearchLocationModelState -> PrestoDOM (Effect Unit) w
+findPlacesIllustration push state =
+  linearLayout
+    [ height MATCH_PARENT
+    , width MATCH_PARENT
+    , orientation VERTICAL
+    , visibility if state.findPlaceIllustration then VISIBLE else GONE
```

Comment:
We can use `length state.locationList == 0`  to trigger this view as it would be only visible when locationList size is zero

Reply:
we need to remove when the user is searching as well

Reply:
For that you can use the variable which is there to trigger the Lottie?